README.md

```diff
@@ -1 +1,51 @@
+
+Welcome to the Rocketpool Community Dune SQL Queries repository! This repository is dedicated to storing and managing SQL queries utilized by the Rocketpool Community, primarily sourced from Dune Analytics.
+
+## Purpose
+
+The purpose of this repository is to centralize and organize SQL queries that are frequently used or modified by members of the Rocketpool Community for data analysis and reporting purposes. By having a centralized location for these queries, it facilitates collaboration, version control, and ensures consistency in analysis across the community.
+
+## Contributing
+
+We welcome contributions from members of the Rocketpool Community to enhance and maintain the repository. If you have queries that you believe would be valuable to add, or if you identify improvements or fixes to existing queries, please feel free to contribute by following these steps:
+
+1. Fork the repository to your own GitHub account.
+2. Clone the forked repository to your local machine.
+3. Create a new branch for your changes: `git checkout -b feature/new-query`.
+4. Make your changes and ensure that the queries are correctly formatted and logically organized.
+5. Test your queries thoroughly using your personal Dune Analytics account to ensure accuracy.
+6. Once satisfied with your changes, commit them:
+`git add`
+`git commit -m "Add new queries for [purpose]"`.
+7. Push your changes to your forked repository: `git push origin feature/new-query`.
+8. Open a pull request (PR) against the main repository, describing the changes and their purpose clearly.
+9. Wait for review and address any feedback or comments provided.
+10. Once approved, your changes will be merged into the main repository.
+
+## Query Structure
+
+Queries in this repository are organized into logical base primitives, which are then combined using SQL `UNION` statements to form larger, more comprehensive queries for broader analysis. This organization helps in modularizing the queries, making them easier to manage, understand, and reuse.
+
+## Usage
+
+To use the queries from this repository:
+
+1. Clone the repository or download the necessary SQL files.
+2. Open the SQL files using your preferred SQL client or editor.
+3. Copy and paste the queries into your SQL environment (e.g., Dune Analytics).
+4. Execute the queries and analyze the results.
+
+## Versioning
+
+This repository follows semantic versioning for releases. Any significant updates or changes to queries will be reflected in the version number according to the rules of semantic versioning.
+
+## License
+
+This repository is provided under the [MIT License](LICENSE), granting permissions for use, modification, and distribution of the content within, with proper attribution.
+
+## Contact
+
+If you have any questions, suggestions, or concerns regarding this repository, feel free to contact us by opening an issue on GitHub or by joining the conversation in the [Rocket Pool Discord](https://discord.gg/rocketpool).
```

Comment:
@acontry thanks for the feedback!  How does this look?

Reply:
Looks great 👍